docs/content/docs/guide/cli.md

```diff
@@ -124,9 +124,13 @@ option to install a webhook forwarder called
 [gosmee](https://github.com/chmouel/gosmee). This forwarder enables connectivity
 between the Pipelines-as-Code controller and GitHub without requiring an
 internet connection. In this scenario, it will set up a forwarding URL on
-<https://hook.pipelinesascode.com> and set it up on GitHub. For OpenShift, it
-will not prompt you unless you explicitly specify the `--force-gosmee` flag
-(which can be useful if you are running [OpenShift Local](https://developers.redhat.com/products/openshift-local/overview) for instance).
+<https://hook.pipelinesascode.com> and set it up on GitHub.
+
+**Important:** On OpenShift, the bootstrap command automatically detects and uses
```

Comment:
Usually, I want to make sure we only use "Important" and things like that sporadically where there is a tricky part of the flow.

We usually add those Important tag when updating documentation because it seems important to say  at the time of a pull request and in that context, but with multiple pull requests piling up, we end up with a lot of "Important" notes everywhere. I'd rather have a proper paragraph that reads from top to bottom like a user would do.